### PR TITLE
Fixed #34554 -- Fixed Reverse(Value(…)) crash on Oracle.

### PR DESCRIPTION
**Pull request by felixxm**
ticket-34554

### CODE REVIEW

File: django/db/models/functions/text.py [hunk 1]
```diff
@@ -268,6 +268,7 @@ def as_oracle(self, compiler, connection, **extra_context):
             ),
             **extra_context,
         )
+        return sql, params * 3
```
Comment (shangxiao) [inline django/db/models/functions/text.py:271]: 👍 This looks correct though am wondering why it was only failing with Value() 🤔

**Review by shangxiao — APPROVED** [review]

**Comment by felixxm** [comment]
buildbot, test on oracle.

File: django/db/models/functions/text.py (hunk 1 above)
Comment (shangxiao) [reply django/db/models/functions/text.py:271]: ah empty params… that old chestnut.

**Comment by felixxm** [comment]
@shangxiao Thanks for checking :+1:

**Pull request merged**